interfaces/builtin/audio_playback.go

```diff
@@ -69,7 +69,7 @@ const audioPlaybackConnectedPlugAppArmorDesktop = `
 # to read available client side configuration settings. On an Ubuntu Core
 # device those things will be stored inside the snap directory.
 /etc/pulse/ r,
-/etc/pulse/* r,
+/etc/pulse/** r,
```

Comment:
For reference, this is already a part of pulseaudio interface:

https://github.com/snapcore/snapd/blob/master/interfaces/builtin/pulseaudio.go#L62-L63

Reply:
One of the files here was a (dangling) symlink to `/run/pulseaudio-enable-autospawn`. Perhaps this ought to be included as well?